dluhc-component-library/src/components/maps/baseMap.tsx

```diff
@@ -59,6 +61,11 @@ const BaseMap = ({
     );
     map.setTarget(ref.current);
     map.addInteraction(new Draw({ source: source, type: "Polygon" }));
+    source.on("addfeature", async function (evt) {
+      var feature = evt.feature;
+      var coords = (feature?.getGeometry() as Polygon).getCoordinates();
+      let features = await fetchDataset(coords);
```

Comment:
We are fetching the data but unless I'm missing something, we arent doing anything with the data?

Reply:
I'm pushing this before I deal with the data as Sam is dealing with a feature which will pass a specific dataset into the API call